library/public/options.h.in

```diff
@@ -114,6 +114,12 @@ public:
    */
   std::pair<std::string, unsigned int> getClosestOption(const std::string& option) const;
 
+  /**
+   *  Returns true if the option is optional else returns false.
+   *  Throws an options::inexistent_exception if option does not exist.
+   */
+  bool IsOptional(const std::string& option) const;
```

Comment:
should be `isOptional`, with a small `i`

Reply:
done